sdk/storage/Azure.Storage.Blobs/src/Generated/BlobRestClient.cs

```diff
@@ -55,10 +57,10 @@ public static partial class Service
                     {
                         Azure.Response _response = async ?
                             // Send the request asynchronously if we're being called via an async path
-                            await pipeline.SendRequestAsync(_request, cancellationToken).ConfigureAwait(false) :
+                            await pipeline.SendRequestAsync(_request, bufferResponse, cancellationToken).ConfigureAwait(false) :
```

Comment:
Nit - parameters should each have a new line

Reply:
This is generated code, so I think it's ok.